### PR TITLE
Fraunces/fix mouse pointer dead zone (fixes #60)

### LINKED ISSUE #60: Fix mouse pointer "dead zone"
See https://github.com/undercasetype/fraunces-minisite/pull/53#pullrequestreview-351401265

### PR DESCRIPTION
**Pull request by marcianosr**

### CODE REVIEW

File: src/js/main.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import FontFaceObserver from "fontfaceobserver";
 
 const fontTimeOut = 5000; // In milliseconds
 const numberOfStickers = 7; // How many hero-stickers-0x.svg do we have?
-const minStickableY = 60; // Keep stickers inside viewport top
+const minStickableY = 0; // Make sure sticker can go under the header bar which allows a smoother sticker interaction.
```
Comment (RoelN) [inline src/js/main.js]: There doesn't seem to be a need for this variable anymore, right?

File: src/js/main.js [hunk 2]
```diff
@@ -274,6 +274,7 @@ window.onscroll = throttle(() => {
 // Update variables related to the viewport
 const setViewportValues = () => {
 	// Redetermine area stickers can be moved in
+
```
Comment (RoelN) [inline src/js/main.js]: Why the newline? ;-)

File: src/js/main.js (hunk 1 above)
Comment (marcianosr) [reply src/js/main.js]: Indeed thanks. Such stupidity of me to let it live!

File: src/js/main.js (hunk 2 above)
Comment (marcianosr) [reply src/js/main.js]: Whoops, not un purpose!

File: src/js/main.js (hunk 1 above)
Comment (RoelN) [reply src/js/main.js]: _KILL IT!!_

**Pull request merged**